lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -272,8 +370,11 @@ def parse(self, inventory, loader, path, cache=True):
 
         config_zones = self.get_option("regions")
         tags = self.get_option("tags")
+        mandatory_tags = self.get_option("mandatory_tags")
+        exclude_tags = self.get_option("exclude_tags")
```

Comment:
An error could be raised when a tag is present in both `mandatory_tags` and `exclude_tags`.